PowerPointLabs/PowerPointLabs/TextCollection.cs

```diff
@@ -42,30 +42,30 @@ public class TextCollection
 
         # region Crop Lab
         public const string CropLabMenuSupertip =
-            "Use Crop Lab to crop shapes and pictures with ease.";
+            "Use Crop Lab to crop shapes and pictures.";
 
         public const string CropLabSettingsSupertip =
-            "Configure the settings for Crop Lab features.";
+            "Configure the settings for Crop Lab.";
 
         public const string MoveCropShapeButtonSupertip =
             "Crop a picture to a custom shape.\n\n" +
-            "To activate, draw one or more shapes upon the picture to crop, select the shape(s), then click this button.";
+            "To perform this action, draw one or more shapes upon the picture to crop, select the shape(s), then click this button.";

```

Comment:
`To perform this action, indicate area-to-keep by drawing one or more shapes upon the picture to crop, ...`?